tooling/nargo_cli/src/cli/compile_cmd.rs

```diff
@@ -413,11 +413,22 @@ mod tests {
 
                 let width = get_target_width(package.expression_width, None);
 
+                // Figure out if there is an upper bound to repeating the entire optimization step after which the results are stable:
```

Comment:
Could you open up an issue for this

Reply:
Maybe this comment is misleading: this was my way to figure out what value I could use for `MAX_OPTIMIZATION_PASSES`, so in that sense this is done, I just left the code here commented out while I was going back and forth between running this vs placing the loop in a smaller scope. I guess the value has been established to be 3 passes. I can remove this comment to reduce confusion, but I'm not sure what further issue to create for it.

There are sub-issues under https://github.com/noir-lang/noir/issues/6667 

Are you suggesting that we should open an issue to investigate _why_ we need 3 passes?

Reply:
> I guess the value has been established to be 3 passes. I can remove this comment to reduce confusion, but I'm not sure what further issue to create for it.

No I was just confused by the comment and thought you wanted to investigate further and just picked 3 as an arbitrary number for now. Let's remove the comment then.